### ISSUE TITLE
configs

### ISSUE THREAD

**Issue by GaozhengPei** [glin2022/atop#2]
why there is no the config of cifar10

**Comment by glin2022** [comment]
We use the config files from the original repository and train/fine-tune the models by directly overwriting the configuration in the [ipynb files](https://github.com/glin2022/atop/tree/main/Finetuning-AToP).

**Comment by 892446631** [comment]
这里是不是有一些问题，你比较的baseline（Diffpure [Nie]）的那篇净化，人家构造对抗样本是用的全白盒攻击，即在构造对抗样本的时候是将前面的净化模块的梯度也算进去的，但是你这里代码构造对抗样本只计算了分类器得梯度。

Is there an issue here? The baseline you are comparing (Diffpure [Nie]) uses a full white-box attack to generate adversarial examples, meaning the gradients of the preceding purification module are also considered during the attack. However, in your code, when generating adversarial examples, you are only calculating the gradients of the classifier.

**Comment by glin2022** [comment]
Thanks for your attention.
1. In the paper, we use DiffPure as an important baseline under AutoAttacks, a powerful attack that combines both white-box and black-box attacks.
2. In the paper, all the attacks method attack the new system of purifier and classifier, i.e., Attack{classifier(purifier(x))}. Please check in the [evaluate.ipynb](https://github.com/glin2022/atop/blob/main/evaluate.ipynb).

**Comment by 892446631** [comment]
very useful.  I think a tutorial should be written to show other people how to use this GitHub project in the ReadMe.md
